docs/DeveloperGuide.md

```diff
@@ -1225,22 +1226,143 @@ testers are expected to do more *exploratory* testing.
+
+   1. Test case: `addt 1 t/depression`<br>
+      Expected: The tag depression is added to the first patient.
+
+   1. Other incorrect add tag commands to try: `addt 0`, `addt 1 t/`<br>
+      Expected: Error message displayed.
+
+### 7.8 Deleting tags from a patient
+
+1. Deleting tags from a patient
+
+   1. Prerequisites: List all patients using the `list` command. Patient List should not be empty. The patient that is chosen should have tags.
+
+   1. Test case: `deletet 1 t/diabetes`<br>
+      Expected: The tag depression is deleted from the first patient.
+
+   1. Other incorrect delete tag commands to try: `deletet 0`, `deletet 1 t/`<br>
+      Expected: Error message displayed.
+
+### 7.9 Finding patients by tag
+
+1. Finding patients by tag
+
+   1. Prerequisites: List all patients using the `list` command. Patient List should not be empty. Chosen Tag should exist in at least one patient.
+
+   1. Test case: `findt depression`<br>
+      Expected: All patients with the tag depression are shown in the list.
+   
+    1. Other incorrect find tag command to try: `findt`<br>
+      Expected: Error message displayed.
+
+### 7.10 Adding an event to a patient
+
+1. Adding an event to a patient
+
+   1. Prerequisites: List all patients using the `list` command. Patient List should not be empty.
+
+   1. Test case: `adde 1 n/Family Visit d/30-09-2024, 12:00 - 15:00`<br>
+        Expected: The event Family Visit is added to the first patient.
+
+   1. Other incorrect add event commands to try: `adde 0`, `adde 1 n/`<br>
+      Expected: Error message displayed.
+
+### 7.11 Deleting an event from a patient
+
+1. Deleting an event from a patient
+
+   1. Prerequisites: List all patients using the `list` command. Patient List should not be empty. The patient that is chosen should have events.
+
+   1. Test case: `deletee 1 e/1`<br>
+      Expected: The first event of the first patient is deleted.
+
+   1. Other incorrect delete event commands to try: `deletee 0`, `deletee 1 e/`<br>
+      Expected: Error message displayed.
+
+### 7.12 Editing an event for a patient
+
+1. Editing an event for a patient
+
+   1. Prerequisites: List all patients using the `list` command. Patient List should not be empty. The patient that is chosen should have events.
+
+   1. Test case: `edite 1 e/1 n/Papa Birthday Celebration d/20-01-2025`<br>
+      Expected: The first event of the first patient is edited to Papa Birthday Celebration.
+
+   1. Other incorrect edit event commands to try: `edite 0`, `edite 1 e/`<br>
+      Expected: Error message displayed.
+
+### 7.13 Sorting the patient list
+
+1. Sorting the patient list
 
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
-      Expected: Similar to previous.
+   1. Prerequisites: List all patients using the `list` command. Patient List should not be empty.
```

Comment:
```suggestion
   1. Prerequisites: Patient List should not be empty.
```

Actually for sort command, there is no prerequisites. The user can start from a state where only 1 user is shown on UI (e.g. after using `find alex` command) and after running `sort`, PatientSync will sort all existing patients in PatientSync and show all existing patients sorted in the UI. If there are no patients, PatientSync will still show sorted successfully just that no patients will be shown on the UI!